.github/workflows/test-build-rpm.yml

```diff
@@ -156,11 +156,9 @@ jobs:
           from: Opendistro Elasticsearch
           # Optional content type:
           content_type: text/html
-
-
-  Start-ES-Service:
+  Test-Plugins-With-Security:
     needs: [signing-artifacts]
-    name: Start elasticsearch service
+    name: Test-Plugins-With-Security
     runs-on: [self-hosted, Linux, X64, odfe-rpm]
```

Comment:
We can use 2 different self hosted runners, one for with security testing and other for without security testing! Running in the same instance may cause issues

Reply:
you are right.  This is just the short term solution. I have created an issue as followup. There are a coupe of tricks in this short term solution:1), has to run security IT first, 2), two IT jobs share some resources.  3), IT should run at job level. one IT might cause the es service down. I saw this issue in the testing. Has to move KNN testing to last step

Even 2 instances can't solve all of the problems

I don't have a deep dive for the cleanup. I think we should make it as a cleanup default step for every job. 
@gaiksaya do you think if it's easy to make two instances work?

Reply:
Got your point! All IT are running under same ES service. If we want to separate them there can be 2 ways:
1. Create self hosted runner for each plugin IT so that they run independently of each other
2. Clean up host after each IT is run (that may need to install ES again and everything will run serially which is time consuming)

For cleaning up host we can use the same block of [code](https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/323/files#diff-5f5d7533e5ed3c7af26fbc72dd7904d4R343) after each job.

**How it works:** The important thing is to keep the github runner that is installed in the ec2 should be running/listening to jobs. Status can be found [here](https://github.com/opendistro-for-elasticsearch/opendistro-build/settings/actions). It should be either idle or active state. To clean up is basically removing the EBS volume with all the configurations and replacing with new volume created using the given snapshot. The [snapshot] (https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/05fa09d8b5f3c36ad2efbe025288b4ffdee9513d/.github/workflows/test-build-rpm.yml#L369) has github runner already installed in it.

Reply:
Cool. My thought is if there is a way to make this clean up as default step in each job. Let's have a deep dive later

Reply:
As discussion offline, we will keep this short term solution now. use the issue for the long term solution